man/man8/zpool-sync.8

```diff
@@ -30,28 +29,25 @@
 .Dd August 9, 2019
 .Dt ZPOOL-SYNC 8
 .Os
+.
 .Sh NAME
 .Nm zpool-sync
-.Nd Force data to be written to primary storage of a ZFS storage pool and update reporting data
+.Nd force data to be written to primary storage of a ZFS storage pool and update reporting data
```

Comment:
Since this supports multiple pools, maybe: "force data to be written to primary storage for ZFS storage pools and update reporting data"

Do we really need the second bit? maybe: ""force data to be written to primary storage for ZFS storage pools"

If we can't agree on this, we can move on, since this is as existing.

Reply:
Point. Going with "flush data to primary storage of ZFS storage pool"